client/src/components/products/AddProduct.js

```diff
@@ -287,11 +318,20 @@ export default class AddProduct extends Component {
             onChange={this.handleChange}
           />
           {/* Location / TO BE ADDED BY NINETTE */}
```

Comment:
Do you mind removing `/ TO BE ADDED BY NINETTE */` here once done?

Reply:
removed it :)